### ISSUE TITLE
As an admin the "order detail page" should be suitable for non-foodtech orders (displayed differently)

### ISSUE THREAD

**Issue by Atala** [coopcycle/coopcycle-web#4697]
- *the packages should be shown*
- the accept/finish/ect buttons not display
- the send payment link button should be displayed differently and somewhere else
other?

![Image](https://github.com/user-attachments/assets/64c6f9c7-6994-411b-8edc-e66b00da6250)

**Comment by vladimir-8** [comment]
Duplicates https://github.com/coopcycle/coopcycle-web/issues/4523 ?

**Comment by Atala** [comment]
I dont think it is exactly a duplicate as for example on this page you have extra features like "send payment link" and "accept order".
Your issue says it misses "duplicate order" for example.
But your point is interesting though.

So i guess now the question is : should we have
1. two different pages with different set of features, like one for handling the delivery part and one the billing part
2. the same but with two different tabs
3. one master page with all the features (send payment link, duplicate order, ...) and details (like all the drops, packages, ...)

Existing design : [link](https://www.figma.com/design/MOAqusOtLKGstcDcTpPIRw/3-I-Locals-commerces?node-id=394-5667&node-type=canvas&t=KlRV2Nxd2BvOZwSd-0) - which does not answer to this question because i guess the designers were ignoring the "existing order" part

so i guess we need to decide that before moving on forward

@Paul-Eraman-CoopCycle

**Comment by Atala** [comment]
we should merge the 2 issues with all the info in1 I guess?
